lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -29,13 +29,15 @@
                      * @returns {HtmlSnippet} - HTML list for course-wide discussion topics.
                      */
                     getCourseWideDiscussionsHtml: function(courseWideDiscussions) {
-                        var subCategoryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
+                        var entryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
```

Comment:
I think we would go for `subCategoryTemplate`. 
In the inline discussion `Type` can be "entry" or "subcategory", is actually the category type. So "category" means either an "entry" or "subcategory".
I am hesitant because I do not have the root knowledge about this so I will defer to you in this regard what is the right naming convention can be.